helm-v3/005-plugins.md

```diff
@@ -3,6 +3,42 @@
 The plugin model will undergo a few changes in order to make it possible to
 support a broader range of features along a broader range of platforms.
 
+## Standard Plugins
+
+The existing plugin model (based on `exec` to local binaries) will continue to
+be supported.
+
+The current YAML format for `plugin.yaml` will be extended as follows:
+
+```
+name: "last"
+version: "0.1.0"
+usage: "get the last release name"
+description: "get the last release name"
+command: "$HELM_BIN --host $TILLER_HOST list --short --max 1 --date -r"
+# New part:
+platformCommand:
+  - os: linux
+    arch: i386
+    command: "$HELM_BIN list --short --max 1 --date -r"
+  - os: windows
+    arch: amd64
+    command: "$HELM_BIN list --short --max 1 --date -r"
```

Comment:
Platform specific hooks would be useful, as well. For example, to install something different on windows, mac, and linux (for binary based plugins)